feature-crowdloan-impl/src/main/java/jp/co/soramitsu/feature_crowdloan_impl/data/network/api/parachain/ParachainMetadataRemote.kt

```diff
@@ -9,6 +9,23 @@ class ParachainMetadataRemote(
     val paraid: BigInteger,
     val token: String,
     val rewardRate: Double?,
-    val customFlow: String?,
-    val website: String
+    val website: String,
+    val flow: ParachainMetadataFlowRemote?
+)
+
+data class ParachainMetadataFlowRemote(
+    val name: String?,
+    val data: ParachainMetadataFlowDataRemote?
+)
+
+data class ParachainMetadataFlowDataRemote(
```

Comment:
How remote differs from non remote?

Reply:
params prod/devApiUrl and prod/devApiKey are just apiUrl and apiKey params depending on build type - release or debug

Reply:
changed to Map
params prod/devApiUrl and prod/devApiKey are mapped to apiUrl and apiKey params depending on build type - release or debug